services/galleryService/galleryService.go

```diff
@@ -0,0 +1,105 @@
+package galleryService
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type JSONPhotosets struct {
+	Photosets struct {
+		Photoset []struct {
+			ID      string
+			Primary string
+			Secret  string
+			Server  string
+			Farm    int
+			Title   struct {
+				Content string
+			}
+			Description struct {
+				Content string
+			}
+		}
+	}
+}
+
+type Album struct {
+	Title       string
+	Description string
+	Link        string
+	ImageURL    string
+}
+
+var jsonBytes []byte
+var photos JSONPhotosets
+
+func init() {
+
+	if os.Getenv("FLICKR_API_KEY") == "" {
+		log.Println("Environment variable FLICKR_API_KEY is not assigned")
+		return
+	}
+
+	if os.Getenv("FLICKR_USER_ID") == "" {
+		log.Println("Environment variable FLICKR_USER_ID is not assigned")
+		return
+	}
+
+	apiKey := os.Getenv("FLICKR_API_KEY")
+	userID := os.Getenv("FLICKR_USER_ID")
+	hc := http.Client{}
```

Comment:
Could we rename this to httpClient? Although gophers like to use 2-letter variable names, it's not immediately obvious that hc refers to an 'HTTP Client'.